core/gather/gatherers/full-page-screenshot.js

```diff
@@ -103,7 +93,7 @@ class FullPageScreenshot extends FRGatherer {
       mobile: deviceMetrics.mobile,
       deviceScaleFactor: 1,
       height,
-      width: 0, // Leave width unchanged
+      width,
```

Comment:
As far as I can tell, the only benefit we get from this change is capping the width at the max webp size but that will only ever be the case if we emulate a device at that extreme size.

I would prefer to just leave `width: 0` so we affect this as little as possible.

Reply:
Seems like it has no other side effect on calculations. Reverted to `width: 0` that might support Flow tests better.